src/brahma/interface/stdio.cpp

```diff
@@ -2,7 +2,6 @@
 // Created by hariharan on 8/16/22.
 //
 #include <brahma/interface/stdio.h>
```

Comment:
```suggestion
#include <brahma/interface/stdio.h>

```
we have a new line in all other files.